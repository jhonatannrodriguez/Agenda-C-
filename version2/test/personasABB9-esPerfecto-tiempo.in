crearArbolPersonas
esPerfectoArbolPersonasTiempo 10000 2
liberarArbolPersonas

crearArbolPersonas
esPerfectoArbolPersonasTiempo2 10000 2
liberarArbolPersonas

crearArbolPersonas
esPerfectoArbolPersonasTiempo2 16383 2
liberarArbolPersonas

Fin
