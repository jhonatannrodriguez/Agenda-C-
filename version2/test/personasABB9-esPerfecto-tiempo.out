1>2>El árbol de personas NO es perfecto.
Bien.
3>4>5>El árbol de personas NO es perfecto.
Bien.
6>7>8>El árbol de personas es perfecto.
Bien.
9>10>Fin.
